Formula/sfml.rb

```diff
@@ -34,8 +34,13 @@ def install
     # headers that were moved there in https://github.com/SFML/SFML/pull/795
     rm_rf Dir["extlibs/*"] - ["extlibs/headers"]
 
+    rpath = "@loader_path/../lib"
```

Comment:
Could we get a formula function for this? Something like `rpath_lib`?

Reply:
Yea, probably. Would be good to know what the proper analogue of this on Linux is though.

Reply:
https://github.com/Homebrew/brew/pull/11187